core/integration/module_typescript_test.go

```diff
@@ -136,17 +136,17 @@ func (TypescriptSuite) TestInit(ctx context.Context, t *testctx.T) {
 			WithWorkdir("/work").
 			WithDirectory("/work/src", c.Directory()).
 			WithNewFile("/work/src/index.ts", `
-				import { dag, Container, object, func } from "@dagger.io/dagger"
+        import { dag, Container, object, func } from "@dagger.io/dagger"
```

Comment:
I did a global search for tab and replace them with space but it seems I didn't re updated the code after.

Reply:
Please don't mark feedback as resolved. It should be the reviewer upon confirming it. Otherwise it's more difficult to keep track of what we need to check for.

In this case the formatting is still all over the place. Doesn't mean I'm going to block because of it, but it's not resolved either.

Reply:
Yeah I did a global cleanup of the tab, that's why there's that many changes on the formatting